packages/frontend/src/actions/actions.ts

```
@@ -125,69 +160,111 @@ export const sendSelectedTextToNote = async (sdk: FrontendSDK) => {
+      parentRequestId: ctx.request.id,
+      sourceKind: window.location.hash === "#/replay" ? "replay" : "history",
+      label: ctx.request.path,
+    });
   } catch (error) {
-    sdk.window.showToast(`Error adding replay session to note: ${error}`, {
+    sdk.window.showToast(`Error saving response to note: ${error}`, {
       variant: "error",
     });
   }
-};
+};
```

ESLint

[error] Insert `⏎`

(prettier/prettier)